template/package.json

```diff
@@ -4,16 +4,23 @@
   "version": "1.0.0",
   "author": "{{ author }}",
   "private": true,
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
   "engines": {
     "node": ">=6",
     "npm": ">=3"
   },
   "scripts": {
     "build": "cross-env NODE_ENV=production npm run clean && npm run test && npm run compile",
+    "module": "cross-env NODE_ENV=production npm run clean && npm run test && npm run compile:module",
+    "build:parts": "npm run sass && npm run compile",
```

Comment:
I can't see where this script is used. The `sass` script has been removed as it is now part of the webpack build.

Reply:
maybe i'm not up to date to your repo. I will update and see if it's still there. :)